vars/RedHat_8.yml

```diff
@@ -1,4 +1,4 @@
 # SPDX-License-Identifier: MIT
 ---
-__snapshot_packages: ["lvm2", "util-linux"]
+__snapshot_packages: ["lvm2", "util-linux", "snapm", "boom-boot"]
```

Comment:
please also add snapm and boom-boot to the appropriate .ostree/packages-runtime-*.txt files

Reply:
ok.  it is a comma seperated list? currently they all just have:

util-linux-core

I will update to:

util-linux-core,  snapm, boom-boot

Reply:
No - not comma seperated in the .ostree/packages*.txt files - just one package per line

Reply:
got it